### ISSUE TITLE
Error: Cannot use import statement outside a module

### ISSUE THREAD

**Issue by tobiasdiez** [Code-Hex/graphql-codegen-typescript-validation-schema#623]
When used with `graphql-codegen-esm`, the following error is triggered:
```
(node:2159) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use `node --trace-warnings ...` to show where the warning was created)
[FAILED] 
[FAILED]               Unable to load template plugin matching 'typescript-validation-schema'.
[FAILED]               Reason:
[FAILED]                 Cannot use import statement outside a module
[FAILED]             
```
This error occurs in 0.14.1, but 0.12.1 was working well. See update PR https://github.com/JabRef/JabRefOnline/pull/2382

**Comment by Code-Hex** [comment]
@tobiasdiez Thanks for your report!

Do you have any plans to resolve this issue?

Probably, Do I need to modify type in package.json?

**Comment by Code-Hex** [comment]
Could anyone give me a minimal reproduction for this?

**Comment by itpropro** [comment]
Any updates on this? This is a breaking bug for production usage, what's the recommended workaround, just using the older version @Code-Hex ?

**Comment by Code-Hex** [comment]
@itpropro Could you give me a minimal reproduction for this?

**Comment by itpropro** [comment]
> @itpropro Could you give me a minimal reproduction for this? So I can fix this issue

Of course, I thought you already had one. It's basically happening when `graphql-codegen-esm` is used with this plugin. I will try to create a stackblitz repro asap!

**Comment by Code-Hex** [comment]
@itpropro Thanks! I check this

**Comment by itpropro** [comment]
> @itpropro Thanks! I check this

Thank you for quickly looking into this :)

**Comment by Code-Hex** [comment]
I just published [v0.16.0](https://github.com/Code-Hex/graphql-codegen-typescript-validation-schema/releases/tag/v0.16.0) which is included this fix.

Big thanks @itpropro

**Comment by tobiasdiez** [comment]
@Code-Hex Thank you very much. I can confirm this is fixed! 😃
